frontend/src/components/ContributionMessages/ContributionMessagesListItem.vue

```diff
@@ -4,9 +4,12 @@
       <b-row class="mb-3 border border-197 p-1">
         <b-col cols="10">
           <small>{{ $d(new Date(message.createdAt), 'short') }}</small>
-          <div class="font-weight-bold" data-test="username">
+          <div class="font-weight-bold" data-test="username" v-if="isNotModerator">
```

Comment:
does this include an Admin too?

Reply:
Yes, the code checks that the logged in user first name and last name is identical with message user first name and user last name. I simply used something already existing. I personally would prefer to check the user id rather than first and last name.